eos/form-factors/baryonic.cc

```diff
@@ -39,21 +39,11 @@ namespace eos
     const constexpr double LambdaBToLambda::mR2_0p;
     const constexpr double LambdaBToLambda::mR2_1m;
     const constexpr double LambdaBToLambda::mR2_1p;
-    const eos::SzegoPolynomial<5> LambdaBToLambda::orthonormal_polynomials(SzegoPolynomial<5>::FlatMeasure(3.221984414));
+    const SzegoPolynomial<5> LambdaBToLambda::orthonormal_polynomials_v(SzegoPolynomial<5>::FlatMeasure(3.221984414));
+    const SzegoPolynomial<5> LambdaBToLambda::orthonormal_polynomials_a(SzegoPolynomial<5>::FlatMeasure(3.475026387));
```

Comment:
Hm. I think this is problematic. It does not correspond to what we did in BMRvD:2022A. Can you make sure that there is an option in the class that only uses the different set of polynomials if activated?

Reply:
Since the `t+_v/a` are now parameters, you just have to set `t+_a = t+_v = whatever you used in bmrvd2022`  as I did and commented in the test case.

Reply:
You are correct sorry, there is a missing piece here

Reply:
It should be better now. Let me know if you like it or not!